library/core/src/num/error.rs

```diff
@@ -113,8 +113,9 @@ pub enum IntErrorKind {
 impl ParseIntError {
     /// Outputs the detailed cause of parsing an integer failing.
     #[must_use]
+    #[rustc_const_unstable(feature = "const_int_from_str", issue = "none")]
```

Comment:
Please update this with a tracking issue number (I think reusing the same one as for parse_str is fine).